solitude/base.py

```diff
@@ -151,6 +152,13 @@ def mozilla_error(self, content, field):
     def braintree_error(self, content, field):
         return [f['code'] for f in content['braintree'][field]]
 
+    def print_as_json(self, data):
+        """
+        A utility function used to dump the json to stdout. Useful in tests
+        when you want some output for documentation.
+        """
+        print json.dumps(data, indent=2, cls=JSONEncoder)
```

Comment:
Curious, why is flake8 complaining about JSON? You would have to convert it to a dict, or have it as a string?